src/assets/js/sfu/sfu.js

```diff
@@ -135,6 +138,9 @@ export default class SFU extends EventEmitter {
         var video = document.getElementById(consumer._appData.peerId + "-video");
         if (video == undefined) {
             video = helper.addVideo(consumer._appData.peerId);
+            if(consumer._appData.videoMuted) {
+                video.parentElement.hidden = true;
```

Comment:
This is the logic to hide the video widget. It will unhide when the mute button is pressed to unmute and the video appears